Gemfile.lock

```diff
@@ -463,4 +487,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   2.1.4
+   2.3.7
```

Comment:
The bundler version was also downgraded in https://github.com/chef/chef/pull/13466/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR466, which wasn't needed. So putting it back to the version it was using.